### PR TITLE
Rework font-size + color

### PR DESCRIPTION
**Pull request by stef74**
related #1376 

First rework on font-size : 
- simplify font-size (global approche for tablet/phone avoid some specific
- remove pixel usage

Your now two standard value : 
- 14px (in desktop) for text
- 0.9em for Card
And some exception (but less than previous)

Some rework is need for Icon ... but not in this PR

 Some size are little changed for : 
- Sometimes some texte are little toot big without very good reason
- Some value are very similar

### CODE REVIEW

File: less/c2corg_ui.less [hunk 1]
```diff
@@ -33,8 +33,16 @@ html, body{
...
   background-color: @body-color;
+  @media @phone {
+    font-size: 0.9em;
+  }
+  @media @tablet {
+    font-size: 0.95em;
+  }
 }
 
+
```
Comment (brunobesson) [inline less/c2corg_ui.less]: why new empty lines?

Comment (stef74) [inline less/c2corg_ui.less]: to be fixe

**Comment by stef74** [comment]
I have define a font-size based on pixel only in body, and so i Have converted old px value to em value.

**Comment by brunobesson** [comment]
@stef74 what's the status of this PR? You do no seem to have made change since a week. Are you stil working on it?

**Comment by stef74** [comment]
No I need to rework on it ... but probably need to rebase before

**Comment by stef74** [comment]
I have tested on many page ... not observed regression concerning font size (tested in split screen with actual master ... and my branch)

**Comment by stef74** [comment]
Ho my god ....

**Comment by brunobesson** [comment]
:rofl:

**Comment by stef74** [comment]
Nothing happen 😄

**Comment by stef74** [comment]
Color are sometimes managed by naming ... so not redereing at the same on different browser / OS.

Replace by RGB code needed.

**Comment by stef74** [comment]
I have to use variable ... work on it this rainy we.

**Comment by stef74** [comment]
Now card have the same size in all page.
I have tested ... but if someone can checkout and test. 
Some optimization can bed added (all color are not managed by variable ... but we can do progresssivly)

**Comment by stef74** [comment]
@brunobesson @asaunier ready for review. I squash after

**Comment by asaunier** [comment]
I don't have an opinion about the CSS changes themselves. I don't know much about CSS.

Concerning the rendering in my Chrome 56, it looks nice (including cards) except that:
* the "filter on map extent" text on the advanced search map became unnecessarily bigger
* the font-size of document texts are now much smaller than cards contents or figures texts => that's not very consistent, font-sizes should be quite the same:

capture d ecran 2017-02-08 a 22 36 14

**Comment by stef74** [comment]
@asaunier many issue have been fixed including font-size of profile feed.
Could you test again if is correct for you now

**Comment by asaunier** [comment]
I don't have a strong opinion about this PR: most changes are quite light.

Document texts and are a bit smaller: why not.
On the other hand I have noticed the orange section titles were a bit bigger: that's not very useful and consistent.
Card texts and titles are a bit smaller, titles are a bit darker: why not.
Figures attribute titles are gray instead of black: why not.

I have noticed the simple search and association tool typed texts are now light grey instead of black:   black was better.

**Comment by stef74** [comment]
According to you remark we propose this fix : 
- [ ] section title 
- [ ] filter on map extent (could you show a screenshoot ...)
- [ ] incrase size of button text in non connected homepage
- [ ] fix color in search tool

**Comment by stef74** [comment]
Other issue on font-size

**Comment by stef74** [comment]
Ok some issue are fixed

**Comment by stef74** [comment]
@asaunier @brunobesson ok for reviewing

**Comment by asaunier** [comment]
I have not tested the branch but feel free to merge if ready.

**Pull request merged**